api-docs/cloud-dedicated/ref.yml

```diff
@@ -0,0 +1,2060 @@
+components:
+  parameters:
+    TraceSpan:
+      description: OpenTracing span context
+      example:
+        baggage:
+          key: value
+        span_id: "1"
+        trace_id: "1"
+      in: header
+      name: Zap-Trace-Span
+      required: false
+      schema:
+        type: string
+  responses:
+    AuthorizationError:
+      content:
+        application/json:
+          examples:
+            tokenNotAuthorized:
+              summary: Token is not authorized to access a resource
+              value:
+                code: unauthorized
+                message: unauthorized access
+          schema:
+            properties:
+              code:
+                description: |
+                  The HTTP status code description. Default is `unauthorized`.
+                enum:
+                  - unauthorized
+                readOnly: true
+                type: string
+              message:
+                description: A human-readable message that may contain detail about the error.
+                readOnly: true
+                type: string
+      description: |
+        Unauthorized. The error may indicate one of the following:
+
+          * The `Authorization: Token` header is missing or malformed.
+          * The API token value is missing from the header.
+          * The token doesn't have sufficient permissions to write to the database.
```

Comment:
Is this context specific to writing, or should this say "... sufficient permissions to write to or query the database."?